live-examples/js-examples/expressions-voidoperator.html

```diff
@@ -0,0 +1,15 @@
+<pre>
+<code id="static-js">void function test() {
+  console.log("boo!");
+  // expected output: "boo!"
+}();
+
+try {
+  test();
+}
+catch(e) {
+  console.log(e);
+  // expected output: ReferenceError: test is not defined
+}
+</code>
```

Comment:
Weird, but I guess this is a weird bit of JS :).

Reply:
There is so much weird JS - but we're getting there :)

I think the point was to demo that you could only call as a self-invoking function.